### DISCUSSION TITLE
Osu!Mania and osu!Catch (Android)

### DISCUSSION THREAD

**Discussion by Rystars** [ppy/osu#16814] [Q&A] [answered]
### _**osu!Catch:I can't move the character**_
![IMG_20220207_201207](https://user-images.githubusercontent.com/96302690/152794708-77484786-4a27-4bd1-a13c-20a8a0aa3af9.jpg)

### _**osu!mania: please enlarge the button, my fingers can't fit on the button**_
![IMG_20220207_201256](https://user-images.githubusercontent.com/96302690/152794721-7d587bb3-f715-44be-ac45-89e6e7d81860.jpg)

**Comment by peppy** [comment, marked as answer]
Please search first.

**Reply by Rystars** [reply]
what do you mean?

**Reply by smoogipoo** [reply]
https://github.com/ppy/osu/issues/4201
